code/style.css

```diff
@@ -1,3 +1,164 @@
+html,
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+body {
+  background: #f7dd74;
+  font-family: "Sora", sans-serif;
+  color: black;
+}
+
+h1 {
+  color: #d342c1;
+  font-size: 2em;
+}
+h2 {
+  font-size: 1.4em;
+}
+
+h3 {
+  font-weight: 300;
+  margin: 0.5em;
+  text-transform: capitalize;
+}
+p {
+  font-weight: 300;
+  margin: 0 0.2em 0;
+}
+
+a {
+  position: relative;
+  color: #000;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #000;
+}
+
+a::before {
+  content: "";
+  position: absolute;
+  display: block;
+  width: 100%;
+  height: 2px;
+  bottom: 0;
+  left: 0;
+  background-color: #000;
+  transform: scaleX(0);
+  transform-origin: top left;
+  transition: transform 0.3s ease;
+}
+
+a:hover::before {
+  transform: scaleX(1);
+}
+
+a {
+  color: black;
+}
+
+header {
+  display: grid;
+  justify-content: center;
+}
+
+.chart-container {
+  margin: 2% 20%;
+}
+
+.user-card {
+  display: grid;
+  justify-items: center;
+  margin: 1em;
+}
+
+.user-card img {
+  height: 200px;
+  object-fit: contain;
+  border-radius: 50%;
+  filter: grayscale(50%);
+}
+
+#projects-container {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: repeat(auto-fit, minmax(305px, 1fr));
+  /* width: 100%; */
+  column-gap: 1rem;
+  row-gap: 1rem;
+}
+
+.project-card {
+  width: 90%;
+  background-color: #fff;
+  /* background-color: #bfbfbf; */
+  opacity: 0.4;
+  padding: 0 1em 1em 1em;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
+}
+
+.project-card:hover {
+  opacity: 0.8;
+}
+
+/* ---- tablet ----*/
+@media (min-width: 668px) {
+  h1 {
+    font-size: 4em;
+  }
+
+  h2 {
+    font-size: 2.5em;
+  }
+
+  h3 {
+    font-size: 1.5em;
+  }
+
+  p {
+    font-size: 1.4em;
+  }
+
+  #projects-container {
+    grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
```

Comment:
I'm curious as to why you continued to use one single column on tablets instead of two. It leaves a considerable amount of space to the right of the cards. This is just a very minor detail, though :)

Reply:
Yes, this was my first try out with grid and I tested a lot, but will do it again with 2 columns and see if it gets better. Thanks for the input!